src/main/webapp/WEB-INF/tags/wrapper.tag

```diff
@@ -121,9 +121,12 @@
             <p>
                 <strong class="phase-tag"><spring:message code="public.template.header.phase.tag" /></strong>
                 <span>
+                    <c:set var="smartSurveyParam">
+                        ${pageContext.request.scheme}://${pageContext.request.serverName}${requestScope['javax.servlet.forward.request_uri']}${empty param.client_id ? '' : '?client_id='}${param.client_id}
```

Comment:
How about the other url params?

Reply:
> Only add client_id param as per Krem's comment https://tools.hmcts.net/jira/browse/SIDM-3128?focusedCommentId=672788&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-672788

There's a new requirement to only add client_id